### ISSUE TITLE
Rules of the game not shown

### ISSUE THREAD

**Issue by arimallick** [arimallick/Blackjack#2]
Problem Statement: The website should show rules of the game!!

> Prerequisites: Basics of HTML

1. Go to index.html
2. Add a paragraph tag ``<p>``  ``</p>`` before line 13 --> `<script src="index.js"></script>`
3. Inside the paragraph tag, write the rules of the game.

> Basic rule of blackjack is that you've to make 21 to win. Less than 21 you can draw another card. More than 21 you're out of the game! **Modify this statement in your own words and write inside the paragraph tag.**

GitHub:
1. Comment "Assign me this issue" if you feel like you're up for the challenge!
2. Once assigned, start working on the issue.
3. Commit and create a PR (pull request) with the issue tag (in this case it is ``Solves issue #2``)
4. Once reviewed, it will be merged!

**Comment by Nitinyad** [comment]
Assign me this issue

**Comment by arimallick** [comment]
LGTM
